WordPress/Classes/Models/SiteInformation.swift

```diff
@@ -1,6 +1,15 @@
 struct SiteInformation {
     let title: String
     let tagLine: String?
+
+    /// if title is nil, then the corresponding SiteInformation value is nil
+    init?(title: String?, tagLine: String?) {
+        guard let title = title else {
```

Comment:
We certainly can; the main thinking for this particular case is that, if no valid title is passed, then a value of `SiteInformation` does not make sense, and this would also simplify the logic in the caller.